lib/notifier.js

```diff
@@ -298,11 +300,18 @@ function addItem(item, callback) {
       if (err) {
         return callback(err);
       }
-      apiClient.postItem(data, function (err, resp) {
-        if (typeof callback === 'function') {
-          callback(err, data, resp);
+      if(SETTINGS.enabled){
```

Comment:
I think I prefer to disable as much code as possible which is in line with the change below regarding the access token check. E.g. I could see `buildItem()` expecting an access token and breaking if one isn't provided.